tests/cbmc/proofs/Makefile-template-defines

```diff
@@ -6,7 +6,7 @@ SRCDIR ?= $(abspath $(PROOF_ROOT)/../../..)
 
 # Absolute path to the litani script.
 #
-LITANI ?= $(abspath $(PROOF_ROOT)/../../litani/litani)
+LITANI ?= litani
```

Comment:
This seems odd. Certainly the comment doesn't match any more since this isn't an absolute path. Maybe we just say "Path to litani script?"

Reply:
Let's change this comment to "How to invoke litani.  Use "litani" when litani is in PATH.   Use an absolute path path to litani when litani is in a submodule".

Let's change this comment in the starter kit for others.